avocado/utils/download.py

```
@@ -118,9 +119,10 @@ def url_download_interactive(url, output_file, title="", chunk_size=102400):
     """
     output_dir = os.path.dirname(output_file)
     with open(output_file, "w+b") as open_output_file:
-        with urlopen(url) as input_file:
+        with requests.get(url, stream=True) as response:
```

Ruff (0.14.4)

Probable use of `requests` call without timeout

(S113)